content/blog/posts/devops-daycamp-2015.rst

```diff
@@ -0,0 +1,38 @@
+DevOps DayCamp
+==============
+:date: 2015-10-29
+:author: Amanda Kelner
+:slug: devops-daycamp
+
+In early October, we hosted our second annual DevOps DayCamp to a over eighty
+students, faculty, and community members, acting as a feeder workshop into the
+year long DevOps BootCamp series. Beyond its promotional function, our DayCamp
+provided an interactive workshop for those in attendance.
+
+We offered seminars and interactive programs for both beginners and more
+advanced participants. For the beginners, the sessions focused on fundamentals
+and introductions. We began with an explanation of DevOps and its functions and
+moved into the basics of operating the system, including working with Unix,
+executing programs, and flagging potential problems. Participants also learned
+collaborative techniques featuring IRC and git.
+
+The advanced track provided the opportunity for attendees to interact with a
+number of guest speakers, including Mozilla’s K Lars Lohn and Emily Dunham,
+Facebook’s Mark Callaghan, and IBM’s Spencer Krum, along with members of the
+Open Source Lab staff at Oregon State, including director Lance Albertson,
+full-time head of software development Ken Lett, and student system
+administrator Daniel Takamori.
+
+To wrap up the festivities, we created our first DevOps Panel, a group of
+experts responding to questions about DevOps. Panelists included release
+engineer at the Linux Foundation Trevor Bramwell, computer science undergraduate
+and system administrator Jeffrey Borcean, full-time systems administrator at the
+OSU Open Source Lab Jordan Evans, and Mozilla web developer Mike Cooper.
+
+In the coming weeks, our DevOps BootCamp will hold weekly sessions covering
+basic software development and system administration skills. The goal is to give
+students the skills to contribute to open-source projects. It's free and open to
+the community so spread the word. For more information, see our website at
+`website`_. See you there!
```

Comment:
The text in the `text`_ will be what's highlight, so you could change it to see 
```
`check out our website`_  
```
Then 
```
.. _check out our website: http://devopsbootcamp.osuosl.org
```